### PR TITLE
Fixes missing Crew manifests on various tablets for roles, mostly service

### PR DESCRIPTION
**Pull request by Zergspower**
## About The Pull Request

This adds the Crew manifest that seems to have been missed when they were all added prior to the other jobs

Chaplain
Clown
Mime
Botanist
Cook
Shaft Miner
Bartender

## How This Contributes To The Skyrat Roleplay Experience

The crew should be able to see what crew they're working with instead of having to go to the messaging app to find out.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

</details>

## Changelog

:cl:
fix: fixes the missing crew manifest from: Chaplain, Clown, Mime, Botanist, Cook, Shaft Miner, Bartender
/:cl:

### CODE REVIEW

**Comment by RatFromTheJungle** [comment]
honestly, if you wanna do non-modular, i'd just throw it in here instead https://github.com/Skyrat-SS13/Skyrat-tg/blob/df4cdc3c29ee6c407c24030eab41fc8fded5a580/code/modules/modular_computers/computers/item/tablet.dm#L376

**Comment by Zergspower** [comment]
how is this non compliant when literally every other department did it this way?
image

**Comment by Zonespace27** [comment]
I'm good with that

File: code/modules/modular_computers/computers/item/role_tablet_presets.dm [hunk 1]
```diff
@@ -159,7 +159,7 @@
 	greyscale_colors = "#D99A2E#69DBF3#E3DF3D"
 	starting_programs = list(
 		/datum/computer_file/program/supermatter_monitor,
-		/datum/computer_file/program/crew_manifest, // SKYRAT EDIT ADDITION - Manifests for all crew
+
```
Comment (Zonespace27) [inline code/modules/modular_computers/computers/item/role_tablet_presets.dm]: however you will want to remove all the whitespace

Comment (Zergspower) [reply code/modules/modular_computers/computers/item/role_tablet_presets.dm]: though this did present a problem with existing crew manifests on security and heads.. hmm

**Comment by Zergspower** [comment]
image
 duplication worry isnt an issue apparently

**Review by Tastyfish — APPROVED** [review]
Man they really aggressively made this awkward to modularize short of replacing `/obj/item/modular_computer/tablet/pda/install_default_programs` but it's not really worth it for a single static app.

**Pull request merged**